samples/highcharts/accessibility/internationalization/demo.js

```diff
@@ -0,0 +1,140 @@
+            downloadSVG: 'Last ned som SVG-bilde',
+            title: 'AAPL Aksjekurs'
+        }
+    },
+    de: {
+        lang: {
+            decimalPoint: ',',
+            thousandsSep: '.',
+            loading: 'Daten werden geladen...',
+            months: ['Januar', 'Februar', 'März', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Dezember'],
+            weekdays: ['Sonntag', 'Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag'],
+            shortMonths: ['Jan', 'Feb', 'Mrz', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dez'],
+            exportButtonTitle: "Exportieren",
+            printButtonTitle: "Drucken",
+            printChart: 'Diagramm ausdrucken',
+            viewFullscreen: 'Im Vollbild anschauen',
+            rangeSelectorFrom: "Von",
+            rangeSelectorTo: "Bis",
+            rangeSelectorZoom: "Zeitraum",
+            downloadPNG: 'Download als PNG-Bild',
+            downloadJPEG: 'Download als JPEG-Bild',
+            downloadPDF: 'Download als PDF-Dokument',
+            downloadSVG: 'Download als SVG-Bild',
+            resetZoom: "Zoom zurücksetzen",
+            resetZoomTitle: "Zoom zurücksetzen",
+            title: 'AAPL Aktienkurs'
+        }
+    }
+};
+
+Highcharts.getJSON('https://demo-live-data.highcharts.com/aapl-c.json', data => {
+
+    const chartOptions = {
+        chart: {
+            style: {
+                fontFamily: 'Arial'
+            }
+        },
+
+        rangeSelector: {
+            selected: 1,
+            labelStyle: {
+                fontWeight: 'bold',
+                color: '#000',
+                fontSize: '14px',
+                padding: '10px'
+            }
+        },
+
+        title: {
+            text: 'AAPL Stock Price'
+        },
+
+        xAxis: {
+            labels: {
+                style: {
+                    fontWeight: 'bold',
+                    color: '#000',
+                    fontSize: '12px'
+                }
+            }
+        },
+
+        navigator: {
+            xAxis: {
+                labels: {
+                    style: {
+                        color: '#000'
+                    }
+                }
+            },
+            series: {
+                label: {
+                    enabled: false
+                }
+            }
+        },
+
+        series: [{
+            name: 'AAPL',
+            data: data,
+            tooltip: {
+                valueDecimals: 2
+            }
+        }]
+    };
+
+    function createChart(options) {
+        return new Highcharts.stockChart('container', options);
+    }
+
+    let chart = createChart(chartOptions);
+    const initialOptions = JSON.parse(JSON.stringify(Highcharts.getOptions()));
+
+    function resetOptions() {
+        Highcharts.setOptions(initialOptions);
+    }
+
+    document.getElementById('en').addEventListener('click', function () {
+        chart.destroy();
```

Comment:
I wonder in this case, if the same options are always present, you might be able to get away with just calling `setOptions` with the new options, and not destroy & recreate the chart.

Disregard this if it doesn't work...